recipes/unittest2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unittest2" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "22882a0e418c284e1f718a822b3b022944d53d2d908e1690b319a9d3eb2c0579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no-argparse-req.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python <3
+    - pip
+    - six
+    - traceback2
```

Comment:
Are these needed at build time?

Reply:
Yes, because otherwise pip tries to download them and we get the no-downloads-allowed error.

Reply:
Weird. Would have thought `--no-deps` would have prevented this issue. :/

Reply:
It seems to arise because something is causing `python setup.py egg_info` to be run? I don't know how the details of this stuff works so I don't know what to make of that.

Reply:
Ah! It's because of the patch. :) Yeah, let's just leave these here then.